src/explore-education-statistics-frontend/src/modules/find-statistics/__tests__/PublicationReleasePage.test.tsx

```diff
@@ -243,7 +243,9 @@ describe('PublicationReleasePage', () => {
     );
 
     userEvent.click(
-      screen.getByRole('button', { name: 'See all updates (2)' }),
+      screen.getByRole('button', {
+        name: `See all updates (2) for ${testRelease.publication.title}`,
```

Comment:
Would hardcode the expected string rather than use a variable. If this variable isn't set correctly, then the test would still pass.